### PR TITLE
TBS Add email sanitization policy to notif config guide

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adds the email sanitization policy to the config email notif guide.
- **Is this PR related to a Monolith release?** No.

### Netlify links
- 
- 

### Resolves

* [OKTA-483921](https://oktainc.atlassian.net/browse/OKTA-483921)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 1]
```diff
@@ -74,41 +74,63 @@ The following table provides a list of all available email templates in an Okta
...
+| Authenticator Reset (Identity Engine)</br></br>MFA Factor Reset (Classic Engine) | Security method reset (Identity Engine)</br>MFA Factor Reset (Classic Engine) | `AuthenticatorReset` | Either `${request.factors}` or `${request.authenticators}` (Identity Engine)</br></br>`${request.factors}`, `${request.date}`, `${request.time}`, and `${request.location}` (Classic Engine) | Sent to users when authenticators are reset (Identity Engine)</br></br>Sent to users when MFA factors are reset (Classic Engine) |
+| Campaign Launched | Access certification campaign: (`campaignName`) | `IGAReviewerNotification` | |   |
+| Campaign Ended | Access certification campaign: (`campaignName`) | `IGAReviewerEndNotification` | |   |
+| Campaign Reminder | Access certification campaign: (`campaignName`) | `IGAReviewerPendingNotification` | |   |
+| Reassigned Review | Access certification campaign: (`campaignName`) | `IGAReviewerReassigned` | |   |
+
+### Use allowed attributes
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:114]: Sure thing. I'll make the change.

**Review by albertchen-okta — APPROVED** [review]
🚢

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 2]
```diff
@@ -74,41 +74,63 @@ The following table provides a list of all available email templates in an Okta
...
+| Email Factor verification (Identity Engine)</br></br>Email Verification (Classic Engine) | Confirm your email address | `EmailFactorVerification` | One of `${verificationLink}`, `${verificationToken}`, `$[registrationEmailVerificationLink}`, or `${registrationEmailVerificationToken}` (Identity Engine) |  Sent to users who must follow the provided link to verify their email address |
+| Forgot Password Denied | Account Password Reset | `ForgotPasswordDenied` | |  Sent to users who forgot their password but only their admin can reset it |
+| Password Reset by Admin | Account Password Reset | `PasswordResetByAdmin` | |  Sent to users who had their password reset by an Okta system administrator, and must follow the provided link to complete the password reset process |
+| Active Directory Password Reset Denied | Forgotten Password | `ADForgotPasswordDenied` | |  Sent to Active Directory users who tried to reset their Active Directory password reset but don't have permission |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:91]:
```suggestion
| Active Directory Password Reset Denied | Forgotten Password | `ADForgotPasswordDenied` | |  Sent to Active Directory users who tried to reset their Active Directory password but don't have permission |
```

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 3]
```diff
@@ -74,41 +74,63 @@ The following table provides a list of all available email templates in an Okta
...
+| Email Factor verification (Identity Engine)</br></br>Email Verification (Classic Engine) | Confirm your email address | `EmailFactorVerification` | One of `${verificationLink}`, `${verificationToken}`, `$[registrationEmailVerificationLink}`, or `${registrationEmailVerificationToken}` (Identity Engine) |  Sent to users who must follow the provided link to verify their email address |
+| Forgot Password Denied | Account Password Reset | `ForgotPasswordDenied` | |  Sent to users who forgot their password but only their admin can reset it |
+| Password Reset by Admin | Account Password Reset | `PasswordResetByAdmin` | |  Sent to users who had their password reset by an Okta system administrator, and must follow the provided link to complete the password reset process |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:90]:
```suggestion
| Password Reset by Admin | Account Password Reset | `PasswordResetByAdmin` | |  Sent to users who had their password reset by an Okta system administrator and must follow the provided link to complete the password reset process |
```

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 4]
```diff
@@ -74,41 +74,63 @@ The following table provides a list of all available email templates in an Okta
...
+| Password Changed | Password Changed | `PasswordChanged` | |  Sent to users whenever their account password changes |
+| Self-Service Unlock when Account is not Locked | Unlock Account | `SelfServiceUnlockOnUnlockedAccount` | |  Sent to users who tried to use self-service to unlock an account that isn't locked |
+| Active Directory Password Unlock | Unlock Account | `ADSelfServiceUnlock` | Either `${unlockAccountLink}` or `${recoveryToken}` (Identity Engine)</br></br>`${unlockAccountLink}` (Classic Engine) |  Sent to Active Directory users who must follow the provided link to unlock their password |
+| Self-Service Unlock Account | Unlock Account | `SelfServiceUnlock` | Either `${unlockAccountLink}` or `${recoveryToken}` (Identity Engine)</br></br>`${unlockAccountLink}` (Classic Engine) |  Sent to users who must follow the provided link to complete self-service unlock account process |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:99]:
```suggestion
| Self-Service Unlock Account | Unlock Account | `SelfServiceUnlock` | Either `${unlockAccountLink}` or `${recoveryToken}` (Identity Engine)</br></br>`${unlockAccountLink}` (Classic Engine) |  Sent to users who must follow the provided link to complete the self-service unlock account process |
```

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 5]
```diff
@@ -74,41 +74,63 @@ The following table provides a list of all available email templates in an Okta
...
+| Campaign Reminder | Access certification campaign: (`campaignName`) | `IGAReviewerPendingNotification` | |   |
+| Reassigned Review | Access certification campaign: (`campaignName`) | `IGAReviewerReassigned` | |   |
+
+### Use allowed HTML tags and elements
+
+The following table provides a list of all allowed HTML tags and elements in customized templates. If you deviate from the allowed tags and elements, you receive error messages.
+
+| HTML tags/elements | Allowed attributes |
+|-------------------|--------------------|
+| All tags | <ul><li>`id`: Can be alphanumeric and contain `:`, `-`, `_`, or `.`</li><li>`class`: Can be alphanumeric and contain `<space>`, `,`, `-`, or `_`</li><li>`lang`: Can be uppercase or lowercase, and 2 to 20 characters long</li><li>`title`: Can contain any letter from any language, any numeric character in any script, and `<space>`, `-`, `_`, `'`, `,`, `:`, `[`, `]`, `!`, `.`, `/`, `\`, `(`, `)`, or `&`</li></ul> |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:120]:
```suggestion
| All tags | <ul><li>`id`: Can be alphanumeric and contain `:`, `-`, `_`, or `.`</li><li>`class`: Can be alphanumeric and contain `<space>`, `,`, `-`, or `_`</li><li>`lang`: Can be uppercase or lowercase, and two to 20 characters long</li><li>`title`: Can contain any letter from any language, any numeric character in any script, and `<space>`, `-`, `_`, `'`, `,`, `:`, `[`, `]`, `!`, `.`, `/`, `\`, `(`, `)`, or `&`</li></ul> |
```

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 6]
```diff
@@ -74,41 +74,63 @@ The following table provides a list of all available email templates in an Okta
...
+| `img` | <ul><li>`src`: Can be one of the following:<ul><li>A string representing an onsite URL containing any letter from any language, any numeric character in any script, or `\`, `dot`, `#`, `@`, `$`, `%`, `+`, `&`, `;`, `-`, `_`, `~`, `?`, `=`, `/`, or `!`</li><li>A URL containing `https`, `ftps`, or `mailto:`</li></ul></li><li>`name`: Can be alphanumeric with `-`, `_`, `$`</li><li>`alt`: Can be a paragraph containing any letter from any language, any numeric character in any script, or `{space}`, `-`, `_`, `'`, `,`, `(`, `)`, `.`, or `;`</li><li>`border-radius`</li><li>`border`</li><li>`hspace`: Must be a number</li><li>`vspace`: Must be a number</li><li>`height`: Can be a number or a percentage</li><li>`width`: Can be a number or a percentage</li><li>`align`: Can be any of `middle`, `center`, `left`, `right`, `justify`, or `char`</li></ul> |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:127]:
```suggestion
| `img` | <ul><li>`src`: Can be one of the following:<ul><li>A string that represents an onsite URL that contains any letter from any language, any numeric character in any script, or `\`, `dot`, `#`, `@`, `$`, `%`, `+`, `&`, `;`, `-`, `_`, `~`, `?`, `=`, `/`, or `!`</li><li>A URL that contains `https`, `ftps`, or `mailto:`</li></ul></li><li>`name`: Can be alphanumeric with `-`, `_`, `$`</li><li>`alt`: Can be a paragraph that contains any letter from any language, any numeric character in any script, or `{space}`, `-`, `_`, `'`, `,`, `(`, `)`, `.`, or `;`</li><li>`border-radius`</li><li>`border`</li><li>`hspace`: Must be a number</li><li>`vspace`: Must be a number</li><li>`height`: Can be a number or a percentage</li><li>`width`: Can be a number or a percentage</li><li>`align`: Can be any of `middle`, `center`, `left`, `right`, `justify`, or `char`</li></ul> |
```

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 7]
```diff
@@ -74,41 +74,63 @@ The following table provides a list of all available email templates in an Okta
...
...~`, `?`, `=`, `/`, or `!`</li><li>`bgcolor`: Can be from the color list or a color code.<ul><li>Color list: `aqua`, `black`, `blue`, `fuchsia`, `gray`, `grey`, `green`, `lime`, `maroon`, `navy`, `olive`, `purple`, `red`, `silver`, `teal`, `white`, or `yellow`</li><li>Color code: Can be HTML/CSS Spec 3 or 6 digit hex</li></ul></li><li>`abbr`: Can be a paragraph containing any letter from any language, any numeric character in any script, or `{space}`, `-`, `_`, `'`, `,`, `(`, `)`, `.`, or `;`</li><li>`axis`: Can be alphanumeric with `-`, `_`, or `$`</li><li>`headers`: Can be alphanumeric with `-`, `_`, or `$`</li><li>`scope`: Must be `row`, `col`, `rowgroup`, or `colgroup`</li><li>`nowrap`</li><li>`height`: Can be a number or a percentage</li><li>`width`: Can be a number or a percentage</li><li>`align`: Can be any of `middle`, `center`, `left`, `right`, `justify`, or `char`</li><li>`valign`: Can be any of `baseline`, `bottom`, `middle`, or `top`</li><li>`charoff`: Can be a number or a percentage</li><li>`char`: Can be any character, and any number of characters (from 0 on), including line terminators</li><li>`colspan`: Must be a number</li><li>`rowspan`: Must be a number</li></ul> |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:129]: not sure why it's not showing my edits in this.....
```suggestion
| `td`</br>`th` | <ul><li>`background`: Can be a string that represents an onsite URL that contains any letter from any language, any numeric character in any script, or `\`, `dot`, `#`, `@`, `$`, `%`, `+`, `&`, `;`, `-`, `_`, `~`, `?`, `=`, `/`, or `!`</li><li>`bgcolor`: Can be from the color list or a color code.<ul><li>Color list: `aqua`, `black`, `blue`, `fuchsia`, `gray`, `grey`, `green`, `lime`, `maroon`, `navy`, `olive`, `purple`, `red`, `silver`, `teal`, `white`, or `yellow`</li><li>Color code: Can be HTML/CSS Spec 3 or 6 digit hex</li></ul></li><li>`abbr`: Can be a paragraph that contains any letter from any language, any numeric character in any script, or `{space}`, `-`, `_`, `'`, `,`, `(`, `)`, `.`, or `;`</li><li>`axis`: Can be alphanumeric with `-`, `_`, or `$`</li><li>`headers`: Can be alphanumeric with `-`, `_`, or `$`</li><li>`scope`: Must be `row`, `col`, `rowgroup`, or `colgroup`</li><li>`nowrap`</li><li>`height`: Can be a number or a percentage</li><li>`width`: Can be a number or a percentage</li><li>`align`: Can be any of `middle`, `center`, `left`, `right`, `justify`, or `char`</li><li>`valign`: Can be any of `baseline`, `bottom`, `middle`, or `top`</li><li>`charoff`: Can be a number or a percentage</li><li>`char`: Can be any character, and any number of characters (from 0 on), including line terminators</li><li>`colspan`: Must be a number</li><li>`rowspan`: Must be a number</li></ul> |
```

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 8]
```diff
@@ -74,41 +74,63 @@ The following table provides a list of all available email templates in an Okta
...
+| `tr` | <ul><li>`background`: Can be string representing an onsite URL that contains any letter from any language, any numeric character in any script, or `\`, `dot`, `#`, `@`, `$`, `%`, `+`, `&`, `;`, `-`, `_`, `~`, `?`, `=`, `/`, or `!`</li><li>`bgcolor`: Can be from the color list or a color code.<ul><li>Color list: `aqua`, `black`, `blue`, `fuchsia`, `gray`, `grey`, `green`, `lime`, `maroon`, `navy`, `olive`, `purple`, `red`, `silver`, `teal`, `white`, or `yellow`</li><li>Color code: Can be HTML/CSS Spec 3 or 6 digit hex</li></ul></li><li>`height`: Can be a number or a percentage</li><li>`width`: Can be a number or a percentage</li><li>`align`: Can be any of `middle`, `center`, `left`, `right`, `justify`, or `char`</li><li>`valign`: Can be any of `baseline`, `bottom`, `middle`, or `top`</li><li>`charoff`: Can be a number or a percentage</li><li>`char`: Can be any character, and any number of characters (from 0 on), including line terminators</li></ul> |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:130]:
```suggestion
| `tr` | <ul><li>`background`: Can be a string that represents an onsite URL that contains any letter from any language, any numeric character in any script, or `\`, `dot`, `#`, `@`, `$`, `%`, `+`, `&`, `;`, `-`, `_`, `~`, `?`, `=`, `/`, or `!`</li><li>`bgcolor`: Can be from the color list or a color code.<ul><li>Color list: `aqua`, `black`, `blue`, `fuchsia`, `gray`, `grey`, `green`, `lime`, `maroon`, `navy`, `olive`, `purple`, `red`, `silver`, `teal`, `white`, or `yellow`</li><li>Color code: Can be HTML/CSS Spec 3 or 6 digit hex</li></ul></li><li>`height`: Can be a number or a percentage</li><li>`width`: Can be a number or a percentage</li><li>`align`: Can be any of `middle`, `center`, `left`, `right`, `justify`, or `char`</li><li>`valign`: Can be any of `baseline`, `bottom`, `middle`, or `top`</li><li>`charoff`: Can be a number or a percentage</li><li>`char`: Can be any character, and any number of characters (from 0 on), including line terminators</li></ul> |
```

**Review by susanharper-okta — APPROVED** [review]
couple nits. approving not to hold you up.

**Pull request merged**